routes/course.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const {
+  getAllCourse,
+  getACourse,
+  createCourse,
+  updateCourse,
+  deleteCourse,
+} = require("../controllers/course");
+const { authenticate } = require("../middlewares/auth");
+const { CourseCreationValidation } = require("../validations/course.validation");
+
+const courseRouter = express.Router();
+
+courseRouter.get("/", authenticate, getAllCourse);
+courseRouter.get("/:id", authenticate, getACourse);
+courseRouter.post("/", CourseCreationValidation, createCourse);
+courseRouter.patch("/", UserUpdateValidation, updateCourse);
```

Comment:
`UserUpdateValidation` is note imported and also rename to  `CourseUpdateValidation`